essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md

```diff
@@ -1,154 +1,208 @@
-  
+
+Issues are generally categorized into either a <b>Feature [FR]</b> or a <b>Bug [Bug]</b>. You can take a look at the description provided for the given issue. 
+
+Send a message down in the comments section to get the issue assigned to yourself. Once that's done, you are ready to begin coding!
+
   ![image](https://github.com/JRS296/AppFlowy-Docs/assets/70965472/a0c1479d-8c1b-4af8-a766-dbff8ab6c3d4)
   *An example of an issue that can be worked on. Remember to message the maintainers first!*
-  
-  <p> As mentioned before, we expect you to go about making contributions using the <a href="https://docs.github.com/en/get-started/quickstart/github-flow">GitHub Flow</a> way. That involves some standard steps: </p>
-  
-1. Creating your own branch within your forked repository
-2. Setting up your development environment
-3. Making changes to the codebase in accordance with the requirements and tasks present in the Issue that you have chosen to work on
-4. Pushing the changes to your branch and creating a Pull Request for the same
-5. Having a code review for the same making further changes
-6. Passing all the automated tests put in place
-7. Handling merge conflicts
-8. Merging your branch, and subsequent deletion of the branch once Merge is successfully completed
 
-<p>These are the steps required to handle your first PR.</p>
+### Working on Your Issue
 
-### How to Propose New Features and Changes for Appflowy
-<p>Congratulations on completing your first PR! Now let us explore how you can propose your ideas for improving AppFlowy with a GitHub Issues walkthrough:</p>
+As mentioned previously, we expect you to make contributions using the <a href="https://docs.github.com/en/get-started/quickstart/github-flow">GitHub Flow</a> workflow.
 
-<b>Step 1:</b> Go to [Appflowy's repository](https://github.com/orgs/AppFlowy-IO/repositories) section on GitHub. Here you can see all of Appflowy's Open-Source repositories that can be contributed to. For the purposes of this article, we shall be looking at the main [Appflowy](https://github.com/AppFlowy-IO/AppFlowy) repository.
-<b>Step 2:</b> Click on the <b>New Issue</b> section of the repository. The subsequent page should look something like this:
+That means following these standard steps:
+
+1. Create your own branch within your forked repository
+2. Set up your development environment
+3. Make changes to the codebase in accordance with the requirements and tasks present in the *Issue* that you have chosen to work on
+4. Push the changes to your branch and creating a Pull Request for the same
+5. Ask for a code review for your PR and make changes as required
+6. Pass all the automated tests that have been put in place
+7. Handle merge conflicts
+8. Merge your branch, and delete of the branch once the merge is successfully completed
```

Comment:
8. Wait for approval from the AppFlowy team, once approved then merge your branch
9. Delete your branch once the PR has been merged